websites/R/RadiosNet/presence.ts

```diff
@@ -0,0 +1,124 @@
+const presence = new Presence({
+    clientId: "1221866611611402363",
+});
+const browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum AssetsRadios {
+    Logo = "https://static.radios.com.br/img/logo-radiosnet-512-solid.png",
+}
+
+presence.on("UpdateData", async () => {
+    const presenceData: PresenceData = {
+        largeImageKey: AssetsRadios.Logo,
+        startTimestamp: browsingTimestamp,
+    };
+
+    if (document.location.pathname === "/")
+        presenceData.details = "Página Inicial";
+    else if (document.location.pathname.includes("/favoritos")) {
```

Comment:
To make the code more consistent/concise, destructure `document.location` properties.
```ts
const { pathname, href, search } = document.location;
```